sub/source/docs/sub-hardware.rst

```diff
@@ -0,0 +1,83 @@
+.. _sub-hardware:
+
+==================================
+Default Sub Hardware Configuration
+==================================
+
+While the Sub firmware is currently being generalized to allow more flexibility, such as RC control, re-arrangement of outputs and inputs, etc.,
+current code is targeted for a specific ROV configuration with the following hardware setup:
+
+- Control is via a surface tether from a ground control station via MAVLink commands to the Sub's autopilot telemetry port, usually employing a joystick and multiple buttons for mode changes and peripheral hardware control. Control functions and autopilot outputs for thrusters and peripheral hardware have specific defaults (see :ref:`Sub defaults <sub-defaults>` below)
+- A depth sensor (underwater barometer) is required.
+- Frame configuration is defaulted to Vectored with 6 thrusters.
+
+.. _sub-defaults:
+
+Sub Defaults
+============
+
+* Default :ref:`frame type <FRAME_TYPE>` is VECTORED
+* RC is disabled via :ref:`RC_PROTOCOLS<RC_PROTOCOLS>` = 0 and :ref:`FLTMODE_CH<FLTMODE_CH>` = 0.
+
+MAVLink Overrides
+-----------------
+Since pilot directional control, normally provided by RC in other vehicles is not enabled by default. Sub uses RC overrides via MAVLink, and the channel mapping is fixed:
```

Comment:
```suggestion
ArduPilot vehicles normally use radio control (RC) to provide pilot directional control. Since Sub has RC disabled by default, it instead provides directional control with MAVLink RC overrides, and the channel mapping is fixed:
```

Reply:
thanks